dev-infrastructure/modules/cs.bicep

```diff
@@ -0,0 +1,138 @@
+@description('The location for the PostGres DB')
+param location string
+
+@description('The managed identity name CS will use to interact with Azure resources')
+param clusterServiceManagedIdentityName string
+
+@description('The managed identity CS uses to interact with Azure resources')
+param clusterServiceManagedIdentityPrincipalId string
+
+@description('An optional user ID that will get admin access on the Postgres database')
+param currentUserId string
+
+@description('An optional user principal name that will get admin access on the Postgres database')
+param currentUserPrincipal string
+
+@description('The name of the database to create for CS')
+param csDatabaseName string = 'ocm'
+
+@description('The AKS cluster name where cluster-service is hosted.')
+param aksClusterName string
+
+@description('The namespace where CS will be hosted.')
+param namespace string
+
+var postgresServerName = 'cs-${location}-${uniqueString(resourceGroup().name)}'
+
+resource postgresAdminManagedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
+  name: '${postgresServerName}-db-admin-msi'
+  location: location
+}
+
+module postgres 'postgres/postgres.bicep' = {
+  name: '${deployment().name}-postgres'
+  params: {
+    name: postgresServerName
+    databaseAdministrators: [
+      // add the dedicated admin managed identity as administrator
+      // this one is going to be used to manage DB access
+      {
+        principalId: postgresAdminManagedIdentity.properties.principalId
+        principalName: postgresAdminManagedIdentity.name
+        principalType: 'ServicePrincipal'
+      }
+      // use the current user as DB admin if defined - for dev purposes
```

Comment:
can we add a // TODO: clean up when we release to prod?

Reply:
i removed it entirely from the bicep files.
admin access to the database for dev purposes can now be aquired on-demand via the makefile target `cs-out-of-cluster-db-access`